### PR TITLE
Spanish translation update and error in english string "outer_only"

### PR DESCRIPTION
**Pull request by tadeu2**
outer_only should be Outer brim only, can you correct in pot file too?

### CODE REVIEW

File: localization/i18n/en/OrcaSlicer_en.po [hunk 1]
```diff
@@ -6983,7 +6983,7 @@ msgid "Mouse ear"
 msgstr ""
 
 msgid "outer_only"
-msgstr ""
+msgstr "Outer brim only"
```
Comment (SoftFever) [inline localization/i18n/en/OrcaSlicer_en.po:6986]: Nice catch.

File: localization/i18n/es/OrcaSlicer_es.po
Comment (SoftFever) [inline localization/i18n/es/OrcaSlicer_es.po:1]: 👍

**Review by SoftFever — CHANGES_REQUESTED** [review]
Hi @tadeu2 
Thanks for the PR.
Overall it looks good.
If you can revert the .gitignore changes, we can merge it.

**Comment by tadeu2** [comment]
Feel free to edit all my PR

**Review by SoftFever — APPROVED** [review]

**Comment by SoftFever** [comment]
👍 
Merged.

**Pull request merged**